dll/win32/advapi32/service/eventlog.c

```diff
@@ -655,7 +655,7 @@ ElfChangeNotify(IN HANDLE hEventLog,
 
     RpcTryExcept
     {
-        Status = ElfrChangeNotify(hEventLog, RpcClientId, (DWORD)hEvent);
+        Status = ElfrChangeNotify(hEventLog, RpcClientId, (DWORD_PTR)hEvent);
```

Comment:
Be careful because https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/idl/eventlogrpc.idl;hb=08d30403777c2db122f2292657a46f42d9f59aa2#l87 indicates a ULONG Event, so either that or the other file is wrong.

Reply:
Fixed by @hpoussin